test/providers/provider_codebuild.test.js

```diff
@@ -0,0 +1,93 @@
+const td = require('testdouble')
+
+const providerCodeBuild = require('../../src/ci_providers/provider_codebuild')
+
+describe('CodeBuild Params', () => {
+  afterEach(() => {
+    td.reset()
+  })
+
+  describe('detect()', () => {
+    it('does not run without CodeBuild env variable', () => {
+      const inputs = {
+        args: {},
+        envs: {},
+      }
+      const detected = providerCodeBuild.detect(inputs.envs)
+      expect(detected).toBeFalsy()
+    })
+
+    it('does run with CodeBuild env variable', () => {
+      const inputs = {
+        args: {},
+        envs: {
+          CI: true,
+          CODEBUILD_CI: true,
+        },
+      }
+      const detected = providerCodeBuild.detect(inputs.envs)
+      expect(detected).toBeTruthy()
+    })
+  })
+
+  it('gets correct params', () => {
```

Comment:
Nest this under a `describe` block for `getServiceParams()` since that's the function being tested here.